libc/hdr/float_macros.h

```diff
@@ -9,11 +9,17 @@
 #ifndef LLVM_LIBC_HDR_FLOAT_MACROS_H
 #define LLVM_LIBC_HDR_FLOAT_MACROS_H
 
-#ifndef LIBC_FULL_BUILD
-// Overlay mode
+#ifdef LIBC_FULL_BUILD
+
+#include "include/llvm-libc-macros/float-macros.h"
+
+#else // Overlay mode
+
 #include <float.h>
-#endif // !LLVM_LIBC_FULL_BUILD
 
+// Filling missing macros if any.
 #include "include/llvm-libc-macros/float-macros.h"
```

Comment:
I still suspect this is not needed, depending on compiler version support.

Reply:
Sure, let just remove it.  We can add it back if any bot complains.